parser-typechecker/src/Unison/Server/Backend.hs

```diff
@@ -782,7 +800,7 @@ renderDoc ::
   Rt.Runtime v ->
   Codebase IO v Ann ->
   Reference ->
-  Backend IO [(HashQualifiedName, UnisonHash, Doc.Doc)]
+  IO [(HashQualifiedName, UnisonHash, Doc.Doc)]
```

Comment:
This didn't use `Backend` at all. Removing it made it cleaner to deal with from `Command`.